src/libraries/System.Collections/ref/System.Collections.cs

```diff
@@ -657,6 +657,7 @@ public partial class List<T> : System.Collections.Generic.ICollection<T>, System
         public void RemoveRange(int index, int count) { }
         public void Reverse() { }
         public void Reverse(int index, int count) { }
+        public System.Collections.Generic.List<T> Slice(int index, int count) { throw null; }
         public void Sort() { }
```

Comment:
The approved API in the issue uses start and length rather than index and count.